services/server/server.go

```diff
@@ -546,22 +546,35 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 		return nil, err
 	}
 
-	if err := s.supervisord.UpdateConfiguration(settings); err != nil {
+	if err := s.UpdateConfigurations(); err != nil {
 		return nil, err
 	}
+
 	if isAgentsStateUpdateNeeded(req.MetricsResolutions) {
```

Comment:
Can we include it into `UpdateConfigurations`?

Reply:
I discussed it @AlekSi. We wanted to extract code which doesn't touch request. So I didn't inclue this one

Reply:
That's because we know for sure that SIGHUP does not change metrics resolution

Reply:
ok